### PR TITLE
Release 6.0.0

### PR DESCRIPTION
**Pull request by corranwebster**
Pyface 6.0.0 release branch.  Not ready for merge yet.

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -101,7 +102,7 @@
 # The name of an image file (within the static path) to use as favicon of the
 # docs.  This file should be a Windows icon file (.ico) being 16x16 or 32x32
 # pixels large.
-html_favicon = "et.ico"
+html_favicon = "favicon.ico"
```
[the remark this answers is not in the document]
Comment (corranwebster) [reply docs/source/conf.py]: Hmmm... the correct favicon is displaying, but I am getting a message about a missing favicon when I run the docs.  The favicon should be coming from the Enthought Sphinx Theme.

I will investigate a little.

[the remark this answers is not in the document]
Comment (corranwebster) [reply docs/source/conf.py]: Ah.  Looks like a minor bug in Enthought Sphinx Theme being papered over by explicitly setting the favicon.ico.  Fine for this release, but will make a PR for the Enthought Sphinx Theme and an issue here.

**Comment by corranwebster** [comment]
Once appveyor is happy, I will tag afb2d0e as the release and upload to PyPI.

File: docs/source/conf.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (jwiggins) [reply docs/source/conf.py]: 👍  Thanks for double checking

**Pull request merged**